addons/website_sale_comparison/static/src/js/website_sale_comparison.js

```diff
@@ -79,6 +84,8 @@ var ProductComparison = publicWidget.Widget.extend(VariantMixin, {
      */
     destroy: function () {
         this._super.apply(this, arguments);
+        this.$scrollingElement.off('.o_product_feature_panel'); // TODO remove in master
+        this.$scrollingTarget.off('.o_product_feature_panel');
```

Comment:
Need to adapt the on/off functionality with event listeners, avoid using jquery for this. Also use double quotes everywhere.